rsmtool/reporter.py

```diff
@@ -617,8 +635,10 @@ def create_report(self, config, csvdir, figdir, context="rsmtool"):
             used_trim_max,
             trim_tolerance,
         ) = config.get_trim_min_max_tolerance()
-        min_score = used_trim_min - trim_tolerance
-        max_score = used_trim_max + trim_tolerance
+
+        if used_trim_min and used_trim_max and trim_tolerance:
+            min_score = used_trim_min - trim_tolerance
```

Comment:
otherwise `min_score` and `max_score` are not defined? they are being used below.

Reply:
good catch, replaced with an assert for mypy.